src/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,152 @@
+.navbar {
```

Comment:
Navbar styles are still detected as `navbar.module.scss`. It shouldn't exist anymore and the `navbar.module.scss` file should be detected. Besides, the styles for the desktop `Navbar` are not proper. The height is different, routes placement and padding between elements and from the page sides.